vignettes/benchmarks.Rmd

```diff
@@ -0,0 +1,195 @@
+---
+title: "diseasystore: Benchmarks"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{diseasystore: Benchmarks}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+To showcase the performance of `diseasystore` on different database backends, we include this vignette that summarises
+a simple benchmark:
+A sample dataset is created based on the `datasets::mtcars` dataset. This data is repeated 10 times and given a unique
+ID (the row number of the data).
+
+A simple `diseasystore` is built around this data, with two `FeatureHandlers`, one each for the `cyl` and `vs`
+variables.
+```{r, eval = FALSE}
+DiseasystoreMtcars <- R6::R6Class(
+  classname = "DiseasystoreBase",
+  inherit = DiseasystoreBase,
+  private = list(
+    .ds_map = list("n_cyl" = "mtcars_cyl", "vs" = "mtcars_vs"),
+    mtcars_cyl = FeatureHandler$new(
+      compute = function(start_date, end_date, slice_ts, source_conn) {
+        out <- benchmark_data |>
```

Comment:
Would be good to see how `benchmark_data` is build/where it comes from

Reply:
Done!